proof/refine/AARCH64/ArchAcc_R.thy

```diff
@@ -672,7 +674,7 @@ next
      apply (simp add: plus_one_eq_asid_pool vref_for_level_def pt_bits_left_def)
      apply (rule conjI, simp add: max_pt_level_def)
      apply (clarsimp simp: level_defs bit_simps maxPTLevel_def)
-     apply word_eqI_solve
+     apply (cases "config_ARM_PA_SIZE_BITS_40"; clarsimp?; word_eqI_solve)
```

Comment:
does this need the quotes?

Reply:
A single name that also happens to be an outer syntax name doesn't really need it, but I thought that we do generally put `cases` terms in quotes. Probably one of those things where we're not consistent.

Reply:
Have removed the quotes for consistency with the other ones -- I don't think we'll be adding any operators here in the future, so no point in having them.

Reply:
For what it's worth, in the past I've thought that it was more common for us to do single names without quotes. Looking back, I guess that was more for short variable names, which I would definitely want without quotes. I guess there could be an argument that longer names like here might be a different case, but consistency is nice.